### PR TITLE
Feat: AWS course IAM + Networking

### PR DESCRIPTION
**Pull request by RiceBowlJr**
Need review for that course. Only IAM and networking courses.
Already used with one class this week for the introduction, some TPs to come overtime.

### CODE REVIEW

File: cours/aws-devops/formation-aws-devops-compute.fr.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  - Les autorisations de lancement qui contrôlent les comptes AWS qui peuvent utiliser l'AMI pour lancer les instances
+  - Un mappage de périphérique de stockage en mode bloc qui spécifie les volumes à attacher à l'instance lorsqu'elle est lancée
+
+### Créer une AMI presonnalisée
+
+   - Lancer une instance à partir d'une AMI existante
+   - Personnaliser cette instance
+   - Enregistrer la configuration mise à jour comme AMI personnalisée
+
+![Schéma Utilisation AMI](https://docs.aws.amazon.com/fr_fr/AWSEC2/latest/UserGuide/images/ami_lifecycle.png)
```
[the remark this answers is not in the document]
Comment (RiceBowlJr) [reply cours/aws-devops/formation-aws-devops-compute.fr.md]: @Nandidier

File: cours/aws-devops/formation-aws-devops-iam.fr.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
+# Identity Access Managment (IAM)
+
+### IAM: Présentation
```
[the remark this answers is not in the document]
Comment (RiceBowlJr) [reply cours/aws-devops/formation-aws-devops-iam.fr.md]: I never know... Always get messy

File: cours/aws-devops/intro-osones.fr.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+### Auteurs
```
[the remark this answers is not in the document]
Comment (rguichard) [reply cours/aws-devops/intro-osones.fr.md]: Agreed

**Comment by rguichard** [comment]
Why formation-aws-devops-compute.fr.md is not part of the course ?

Also, I would name the course "aws", no need to add complexity with "cloud-devops"
aws-devops/intro-osones.fr.md should be removed as it is not used.

**Comment by RiceBowlJr** [comment]
I strongly agree with you @rguichard

File: cours/aws-devops/intro-osones.fr.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (RiceBowlJr) [reply cours/aws-devops/intro-osones.fr.md]: Removed.

**Comment by RiceBowlJr** [comment]
I think we can remove compute completly since it's still work in progress. Don't you think @Nandidier ?

File: cours/aws/aws-compute.fr.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
+# AWS Calcul
+
+## EC2
+
+### Qu'est-ce qu'Amazon EC2 ?
+
+ - Elastic Compute Cloud
+ - Offre une capacité de calcul évolutive dans le cloud Amazon Web Services (AWS)
+ - Environnements de calcul virtuels, appelés **instances** (équivalent du serveur ou de la machine virtuelle)
```
Comment (RiceBowlJr) [inline cours/aws/aws-compute.fr.md]: Only two spaces before a `-` and if you want to increment add two more and that goes on and on.

**Pull request merged**